plugins/woocommerce/tests/e2e-pw/tests/merchant/create-woocommerce-blocks.spec.js

```diff
@@ -52,7 +52,7 @@ const test = baseTest.extend( {
 
 test.describe(
 	'Add WooCommerce Blocks Into Page',
-	{ tag: [ '@gutenberg', '@services' ] },
+	{ tag: [ '@gutenberg', '@services', '@skip-on-default-pressable' ] },
```

Comment:
It failed with error:

```
Error: Error getting WordPress version: Command failed: pnpm exec wp-env run tests-cli -- wp core version
ℹ Starting 'wp core version' on the tests-cli container. 

service "tests-cli" is not running
✖ Command failed with exit code 1
Command failed with exit code 1
```

Many tests have failed with this error, and I'm skipping them on Pressable until I find the solution.

Reply:
Ah, got it, I totally forgot about that part. It's where it checks for the WP version, because the insert block pieces changes in 6.6.
We'll need to change the way to check the WP version. Also WP 6.7 is right around the corner, so worst case we'll be able to drop this version check then.